sbin/release.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+VERSION=$(cat package.json | jq .version)
```

Comment:
- [ ]  Seems like current we need to update `package.json` version before running the script. Would be great if we got a prompt for the version or the version is updated based on the conventional commits

Reply:
The idea is that version updates will be based on how a PR was "tagged". With yarn v2 you can indicate the intended version upgrade. When you eventually do the upgrade, it will look at all those tags and find an appropriate new version number.

Reply:
I've added some explanation now to the CONTRIBUTING.md file.